gdb-7.8/gdb/hsail-tdep.c

```diff
@@ -106,6 +106,8 @@ void hsail_linux_initialize_stg2(void)
    * */
 
   struct ui_out* uiout = current_uiout;
+  char * hsaBreakpointLocation = "TriggerStop";
```

Comment:
I think the function name is not consistent with the name in the agent "TriggerGPUBreakpointStop"